lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -196,7 +196,7 @@ abstract class SqlParserTestBase : TestBase() {
      *
      * Verify that the result matches the original without metas.
      */
-    private fun roundTripPigAstToExprNode(actualExprNode: ExprNode) {
+    private fun assertRoundTripPigAstToExprNode(actualExprNode: ExprNode) {
```

Comment:
Resolve the comment, https://github.com/partiql/partiql-lang-kotlin/pull/462#discussion_r740551085